src/libs/registry/docset.cpp

```diff
@@ -647,9 +671,24 @@ static void matchFuzzy(const char *needle, int needleLength,
 
         while (j < haystackLength) {
             if (needle[i] == haystack[j++]) {
-                if (*start == -1)
+                if (*start == -1) {
                     *start = j;  // first matched char
 
+                    // try starting the search later in case the first character occurs again later
+                    int recStart, recLength, recScore;
+                    matchFuzzy(
```

Comment:
Don't break the line immediately after parenthesis, there's quite a lot of space on the same line (up to 100 characters).